avaframe/com1DFAPy/com1DFA.py

```diff
@@ -674,7 +704,7 @@ def computeLeapFrogTimeStep(cfg, particles, fields, dt, dem, Ment, Cres, Tcpu):
     # dtK5 is half time step
     dtK5 = 0.5 * dt
     # cfg['dt'] = str(dtK5)
-    log.info('dt used now is %f' % dt)
+    # log.info('dt used now is %f' % dt)
```

Comment:
Either delete it, or drop the level to debug

EDIT: since you debug log it in the time discret., delete it here...